cumulusci/tasks/salesforce/Deploy.py

```diff
@@ -42,29 +42,56 @@ class Deploy(BaseSalesforceMetadataApiTask):
         },
     }
 
+    # ignore all files in ./lwc and all sub-directories of a lwc component (i.e. ./lwc/[lwc Component Name]/.*)
+    _directory_blacklist = re.compile("^\\.\\/lwc(\\/.*?\\/.*)?$")
```

Comment:
In another comment I'm recommending moving away from regexes anyway, but fwiw in Python you can use a "raw string" to avoid needing to escape backslashes: `r"^\.$"`

Reply:
Applying suggestions in another comment, I will split `root` and do logic  with the parts instead of regex

Reply:
Change is done.  directory/file checking is done in `_include_directory`/`_include_file` respectively